packages/coinstac-client-core/package.json

```diff
@@ -41,7 +41,9 @@
     "commander": "^2.9.0",
     "convict": "^1.5.0",
     "csv-parse": "^1.2.0",
+    "cycle": "^1.0.3",
```

Comment:
Indeed. No idea what that actually does, it’s not required anywhere (57c579675033d1eb3dcd12fd0d790a6f1a87503c).